addons/project/wizard/project_template_create_wizard.xml

```diff
@@ -14,6 +14,8 @@
                 </div>
                 <group>
                     <group>
+                        <field name="date" required="not template_has_dates and date_start" invisible="1"/>
+                        <field name="date_start" placeholder="Today" required="not template_has_dates and date" invisible="1"/>
                         <field name="date_start" string="Planned Date" widget="daterange" options='{"end_date_field": "date", "always_range": "1"}'/>
```

Comment:
i didn't get this change, you added 2 invisible fields but their values weren't used

- date were used in date_start but it's already invisible
- date_start already exist in line 19.

what's the purpose ?

Reply:
It is to set the "required" attribute. The second line is not useful though, I was trying to have the placeholder only on `date_start`, and potentially having to make a custo for it, but we ended up deciding against it